apache/zeppelin
# ZEPPELIN-2984: allow enter from username

### What is this PR for?
Allow an "Enter" keypress in the username field during login

### What type of PR is it?
Improvement

### Todos
* N/A

### What is the Jira issue?
* https://issues.apache.org/jira/browse/ZEPPELIN-2984

### How should this be tested?
* Pull up login screen. Fill in username and password. Click in username field.
  Press enter. We would expect the login handler to be called.
* Pull up login screen. Fill in username and password. Press enter (while still
  in password field). We would expect the login handler to be called just as
  before.
* Alternatively, use a password manager (e.g. LastPass). Save the password.
  Pull up the login screen. Press Enter. We would expect the login handler to be
  called.

### Screenshots (if appropriate)

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? No
